examples/weldon-on-camelyon/sim.py

```diff
@@ -0,0 +1,268 @@
+import argparse
+import os.path
+from collections import OrderedDict
+from typing import Dict, Tuple, List
+from copy import deepcopy
+from pathlib import Path
+
+import torch
+from torch.utils.data import DataLoader
+
+import flwr as fl
+from flwr.common import Metrics
+from flwr.common.typing import Scalar
+
+from datasets.utils.logging import disable_progress_bar
+from substrafl.index_generator import NpIndexGenerator
+
+from dataset_manager import fetch_camelyon, reset_data_folder, creates_data_folder
+from data_managers import Data, CamelyonDataset
+from model import Weldon
+from utils import train, test
+
+
+parser = argparse.ArgumentParser(description="Flower Simulation for weldon on camelyon")
+
+parser.add_argument(
+    "--num-cpus",
+    type=int,
+    default=1,
+    help="Number of CPUs to assign to a virtual client",
+)
+parser.add_argument(
+    "--num-gpus",
+    type=float,
+    default=1.0,
```

Comment:
Proably better to keep it a `0.0` by default, then show in the REAMDE how to use GPU?